chainerrl/agents/dqn.py

```diff
@@ -266,16 +268,18 @@ def update_from_episodes(self, episodes, errors_out=None):
                     transitions.append(ep[i])
                     if has_weights:
                         weights_step.append(weights[index])
-                batch = batch_experiences(transitions,
-                                          xp=self.xp,
-                                          phi=self.phi,
-                                          batch_states=self.batch_states)
+                batch = batch_experiences(
+                    [transitions],
+                    xp=self.xp,
+                    phi=self.phi,
+                    gamma=self.gamma,
+                    batch_states=self.batch_states)
```

Comment:
```suggestion
                    batch_states=self.batch_states)
                assert len(batch['state']) == len(transitions)
```

Reply:
This is how `update_from_episodes` works:
- sample N episodes from `EpisodicReplayBuffer`
- take the first transition of each episode, batch those transitions, and compute a loss on that batch
- take the second transition of each episode, ...
- ...
- take the last transition of each episode, ...
- sum the losses, backprop, and update the model

Thus, `len(batch['state'])` should be equal to the number of transitions taken at each step.